doc/man3/RSA_public_digest.pod

```diff
@@ -0,0 +1,41 @@
+=pod
+
+=head1 NAME
+
+RSA_public_digest, RSA_private_digest - get RSA digests for the public or private keys
+
+=head1 SYNOPSIS
+
+ #include <openssl/rsa.h>
+
+ int RSA_public_digest(RSA *rsa, const EVP_MD *mdtype, unsigned char *md,
+                       unsigned int *mdlen);
+ int RSA_private_digest(RSA *rsa, const EVP_MD *mdtype, unsigned char *md,
+                        unsigned int *mdlen);
+
+=head1 DESCRIPTION
+
+RSA_public_digest() computes a digest of the ASN.1 encoded
+public key as specified by B<mdtype> and deposits the value into B<md>,
```

Comment:
Why not use the `B<RSAPublicKey>` to replace the `public key` phrase? Thus this is consistent with below sentence...